plugins/hlf.yaml

```diff
@@ -3,8 +3,8 @@ kind: Plugin
 metadata:
   name: hlf
 spec:
-  version: "v1.6.2"
-  homepage: https://github.com/kfsoftware/hlf-operator
+  version: "v1.6.3-beta7"
```

Comment:
im pretty sure we don't support versioning like this

Reply:
I'm going to tag it with another version: `v1.6.3`

Not sure if krew-release-bot will create another PR or update this one

Reply:
https://github.com/hyperledger-labs/hlf-operator/actions/runs/2302413715